3574/2/helpers/save.rb

```diff
@@ -0,0 +1,30 @@
+require 'fileutils'
+
+module SaveHelper
+  def save_data(action)
+    make_dir(action)
+    save_photo(action)
+    save_location(action)
+  end
+
+  def make_dir(action)
+    session[:timestamp] = Time.now.strftime('%Y-%m-%d_%H-%M-%S')
+    FileUtils.mkdir_p(path_for(action))
+  end
+
+  def save_photo(action)
+    File.open(path_for(action) + 'photo.jpg', 'wb') do |file|
```

Comment:
we usually use string interpolation instead of string concatenation
looks reasonable to move path_for(action) to separate variable and then use it in string interpolation